index.js

```diff
@@ -11,8 +11,8 @@ function format(obj) {
     : JSON.stringify(obj);
 }
 
-module.exports = function(hash) {
-  function visit(hash, prefix) {
+module.exports = function(hash, indent=0, newline_after_section=false) {
```

Comment:
These should probably be part of an optional `options` object instead of being additional parameters. That way, we can easily add more in the future, and consumers can configure specific options if desired.